sklearn/base.py

```diff
@@ -12,8 +12,18 @@
 from .utils.fixes import signature
 from . import __version__
 
+_DEFAULT_TAGS = {
+    'input_types': ['2darray'],
```

Comment:
yes. Maybe "data_types"? We have 1d, 2d, 3d (patch extractor), sparse, dictionaries and text. Also tuples for dict-vectorizer (or was it hashing vectorizer?) I think.